### ISSUE TITLE
[documentation] Page Structure

### ISSUE THREAD

**Issue by hjusassac** [way-seoul/on-my-way#28]
| no. | pages | users before login | logged-in users | admin users |
|---|---|---|---|---|
| 1 | home | O | O | O |
| 2 | login | O |  |  |
| 3 | register(add-user) | O |  | O |
| 4 | challenge-specific |  | O | O | 
| 5 | admin(= user-list & challenge-list) |  |  | O |
| 6 | edit user |  |  | O |
| 7 | add challenge |  | O | O |
| 8 | edit challenge |  | O (by whom created) | O |
| 9 | user-[my page] |  | O | O |
| 10 |  |  |  |  |
| 11 |  |  |  |  |
| 12 |  |  |  |  |

**Comment by polarbearTW** [comment]
Index.php should be the landing page for the user is not logged in:
-The user will see a map with preview of challenges
-If the user click a challenge, will be directed to log in page
-A button "Play Now", will be directed to log in page

Home.php should be the landing page for the user is logged in
-The user will see a map with challenges
-If the user click a challenge, will be directed to specific challenge page
-A button "Add Challenge", will be directed to add challenge page
-A button "See My profile", will be directed to user profile page
-A button "Log Out", will be directed to the main page

The admin page will not be shown on home page or on the header.
The admin page will be only shown if logged in as admin.
For now, in order to enter admin page, you have to enter action=admin
If the user is logged in as admin:
-The admin will see a list of users and challenges
-The admin will be able to manage users & challenges (add/delete/edit)

**Comment by hjusassac** [comment]
Let's have a preview page for users before login(landing page that Jusy mentioned) 

so on home page, show places near the logged-in user's current location(when access granted)
:: by default, show wcoding location and then pan the map once access granted/location detected

on landing page, show map preview(?) so that we don't need to ask for the current location access!

**Comment by polarbearTW** [comment]
> Let's have a preview page for users before login(landing page that Jusy mentioned)
> 
> so on home page, show places near the logged-in user's current location(when access granted) :: by default, show wcoding location and then pan the map once access granted/location detected
> 
> on landing page, show map preview(?) so that we don't need to ask for the current location access!

Yup, I edited the comment for how each page should be linked (from my understanding). 
Let me know if this is the structure that we want
